src/context/context.interface.ts

```diff
@@ -0,0 +1,24 @@
+import { AuthFormData } from "src/common/interface";
+import { User } from "src/modules/User";
+
+export interface GlobalState {
+  user?: User;
+  loading: boolean;
+}
+
+export enum Types {
+  REQUEST = "REQUEST",
+  SET_USER = "SET_USER",
+  LOADER_OFF = "LOADER_OFF",
+}
+
+export type Dispatch = React.Dispatch<{
+  type: Types;
+  payload?: any;
```

Comment:
Please provide appropriate type

Reply:
I think payload will vary for different actions so can't have any fixed type for this `any` is the most suitable type here.